### PR TITLE
{devel}[GCCcore/8.2.0] GConf v3.2.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/73eeb5586ed5ea417314b15e80a85f7c for a full test report.

File: easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.110-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://dbus.freedesktop.org/releases/dbus-glib']
+sources = [SOURCE_TAR_GZ]
+checksums = ['7ce4760cf66c69148f6bd6c92feaabb8812dee30846b24cd0f7395c436d7e825']
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('pkg-config', '0.29.2'),
+    ('Python', '3.7.2'),  # Python is needed for using GLib at build time.
```
Comment (boegel) [inline easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.110-GCCcore-8.2.0.eb]: `for building against GLib`?

File: easybuild/easyconfigs/g/GConf/GConf-3.2.6-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    'ed55bff5bc239115ae27b8520a923ae0d5cab6cb15ca9921a8c23b939306489f',  # GConf-3.2.6_call-dbus_g_thread_init.patch
+    'f8f7745c3648953098eb56af0f64adc69d8c0b0e78c8db351746a2b4c58e8bb4',  # GConf-3.2.6_fix_wrong_return_value.patch
+    'ee5524c3cb985088bbe4884531ddc48ba22d2fced2a34865304c77e194a55bde',  # GConf-3.2.6_gconf-path-max-hurd.patch
+    # GConf-3.2.6_readd_gconf_engine_key_is_writable.patch
+    'b95dfc2f0474d9162fc686f26cd4d3da5c9b01a9c609dc0a4f2b2bd901d497f3',
+]
+
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('GObject-Introspection', '1.60.1', '-Python-3.7.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GConf/GConf-3.2.6-GCCcore-8.2.0.eb]: @akesandgren Should we list `Python` (explicitly) here too?

Comment (akesandgren) [reply easybuild/easyconfigs/g/GConf/GConf-3.2.6-GCCcore-8.2.0.eb]: Nah, not really necessary...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3113.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/2135553287d17473b4bf52f8d463950f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2594.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4bf02de38e70f8aafdf088a0d34d2a52 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fa9bd95ad5651f7a68d26a8852db0e41 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**